src/main/java/seedu/address/logic/commands/EditClientCommand.java

```diff
@@ -94,9 +96,15 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_NON_EXISTING_PRODUCT);
         }
 
+
         model.setClient(clientToEdit, editedClient);
         model.updateFilteredClientList(PREDICATE_SHOW_ALL_CLIENTS);
-        return new CommandResult(String.format(MESSAGE_EDIT_CLIENT_SUCCESS, editedClient));
+        clientToEdit.getMeetings().forEach(meeting ->
+                model.deleteMeeting(meeting));
+        editedClient.getMeetings().forEach(meeting ->
+                model.addMeeting(meeting));
+        model.updateFilteredMeetingList(PREDICATE_SHOW_ALL_MEETING);
```

Comment:
hmm, why is there a call to update meeting list in an edit client command?

Reply:
I see how this causes dependency issue, will fix by tmrw.